pkg/authorization/client/clientset_generated/internalclientset/clientset.go

```diff
@@ -0,0 +1,73 @@
+package internalclientset
+
+import (
+	"github.com/golang/glog"
+	v1core "github.com/openshift/origin/pkg/authorization/client/clientset_generated/internalclientset/typed/core/v1"
+	restclient "k8s.io/kubernetes/pkg/client/restclient"
+	discovery "k8s.io/kubernetes/pkg/client/typed/discovery"
+	"k8s.io/kubernetes/pkg/util/flowcontrol"
+)
+
+type Interface interface {
+	Discovery() discovery.DiscoveryInterface
```

Comment:
We can potentially punt on discovery via this interface for now.

On Mon, Jun 27, 2016 at 2:42 PM, Michal Fojtik <notifications@github.com>
wrote:

> In
> pkg/authorization/client/clientset_generated/internalclientset/clientset.go
> <https://github.com/openshift/origin/pull/9372#discussion_r68632686>:
>
> > @@ -0,0 +1,73 @@
> > +package internalclientset
> > +
> > +import (
> > +	"github.com/golang/glog"
> > +	v1core "github.com/openshift/origin/pkg/authorization/client/clientset_generated/internalclientset/typed/core/v1"
> > +	restclient "k8s.io/kubernetes/pkg/client/restclient"
> > +	discovery "k8s.io/kubernetes/pkg/client/typed/discovery"
> > +	"k8s.io/kubernetes/pkg/util/flowcontrol"
> > +)
> > +
> > +type Interface interface {
> > +	Discovery() discovery.DiscoveryInterface
>
> @smarterclayton <https://github.com/smarterclayton> will public var
> LegacyPrefix do or we can do smarter plumbing?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9372/files/2a0ceea3c7e61fc7e21b9e5471a651781638e7d5#r68632686>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p2K_flPIX53pO_2nX6BpPDAiaG09ks5qQBmwgaJpZM4I3W73>
> .
>